### PR TITLE
Creates very basic UI for allowing user to create trips

### PR DESCRIPTION
**Pull request by PriyaVarra**
This PR adds a first basic implementation of a UI for users to add trips under their account. There is currently no backend functionality and comments have been placed indicating where it should be added.

If a user is logged in and has toggled on displayed places, the search options get replaced, and they can type in the the text field to add cards for each new trip. Cards can be dragged to trashcan symbol to be deleted. If a trip is clicked, it is emphasized and becomes non-draggable, and only one can be emphasized at a time.

Screenshot 2020-08-05 at 11 10 35 AM

### CODE REVIEW

File: where-to-travel/src/main/webapp/index.html [hunk 1]
```diff
@@ -107,16 +107,41 @@ <h1 class="mb-0">
...
             <button type="button" id="submit" class="btn btn-primary search-btn font-size-large">Search</button>
             <button type="button" onclick="clearSearch()" class="btn btn-primary search-btn font-size-large">Clear</button>
-          </span>
+            
           
-          <div id="feedback-target"></div>
+            <div id="feedback-target"></div> 
+            
+          </div>
+
```
Comment (emmapaczkowski) [inline where-to-travel/src/main/webapp/index.html]: Could you remove the space?

File: where-to-travel/src/main/webapp/index.html [hunk 2]
```diff
@@ -107,16 +107,41 @@ <h1 class="mb-0">
...
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="card" id="trips">
+            </div>
+          </div>
+
         </div>
```
Comment (emmapaczkowski) [inline where-to-travel/src/main/webapp/index.html]: Delete spaces?

File: where-to-travel/src/main/webapp/style.css [hunk 3]
```diff
@@ -176,6 +176,11 @@ input:checked + .slider:before {
   margin-left: auto;
 }
 
+.plus {
+  outline: none;
+  margin-left: 12px;
```
Comment (emmapaczkowski) [inline where-to-travel/src/main/webapp/style.css]: Alphabetize

**Review by andy7809 — APPROVED** [review]
LGTM

File: where-to-travel/src/main/webapp/style.css (hunk 3 above)
Comment (PriyaVarra) [reply where-to-travel/src/main/webapp/style.css]: fixed!

File: where-to-travel/src/main/webapp/index.html (hunk 2 above)
Comment (PriyaVarra) [reply where-to-travel/src/main/webapp/index.html]: fixed!

**Pull request merged**